Formula/gerbil-scheme.rb

```diff
@@ -27,10 +27,37 @@ def install
       gxtags
     ]
 
+    # Find the installation correctly if GERBIL_HOME unset
```

Comment:
Do we need to have `GERBIL_HOME` set for build time or for runtime as well?

For build time I suggest to set `ENV["GERBIL_HOME"] = libexec`
For run time, we could use `env_script_all_files` for setting `GERBIL_HOME`

Otherwise, it looks like quite a heavy patching to me, which we tend to avoid in Homebrew.

Reply:
> Do we need to have `GERBIL_HOME` set for build time or for runtime as well?

Build time, runtime, and runtime of executables generated by Gerbil.